app/controllers/user_sessions_controller.rb

```diff
@@ -0,0 +1,28 @@
+class UserSessionsController < ApplicationController
+  skip_before_action :require_login, only: [:index, :create]
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = login(user_params[:email], user_params[:password])
+
+    if @user 
+      redirect_back_or_to(:users, notice: 'Login successful')
```

Comment:
In general I prefer to use double quotes.

```suggestion
      redirect_back_or_to(:users, notice: "Login successful")
```